app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="text_view_text_size_x_big_35">35sp</dimen>
```

Comment:
try not put the dimen on the name please

Reply:
Done